migra/formatter.py

```diff
@@ -0,0 +1,17 @@
+import re
+
+class Formatter:
+
+    def __init__(self, eager_parens: bool=True) -> None:
```

Comment:
inconsistent return type, either `-> None` both methods or don't

would prefer if you just drop it altogether because its verbose + implied, but at least be consistent